

Title: Mapping of global transcriptional regulator MftR binding loci in Burkholderia thailandensis using ChIP-seq

Name: Ahmed Al-Tohamy

Background information:

The widespread and misuse of antibiotics speeded the selection and evolution of antibiotic resistant bacteria. Among those bacterial species is Burkholderia (B.) which can cause serious diseases to human, animals and plants. Multiple Antibiotic Resistance Regulator (MarR) is a family of bacterial transcription factors that regulates environmental stress, virulence genes, and catabolism of aromatic compounds.Major Facilitator Transport Regulator (MftR) is MarR homolog and B. thailandensis encodes 12 annotated MarR, one of them is mftR. We know that mftR regulate approximately 400 genes in B. thailandensis. The knowledge gap is we don’t know where exactly mftR bind. Also we don’t have a clear picture about mftR pathway and consequently its effect on antibiotic resistant.

Data required:

Currently I am working on the wet lab part of ChIP to send the samples for sequencing. I have searched through NCBI Gene Expression Omnibus (GEO) database and found quite similar dataset under GEO accession (GSE61334). They performed ChIP-seq for the same bacterial species with different strain. Link to download the data: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61334

Questions and corresponding analysis.

    Analysis ChIP-seq data.
    Building Gene regulatory network (GRN) using ChIP-seq and RNA-seq data.
    Better understating of the global transcriptional regulatory mftR function.
    Get a clearer understanding of factors responsible for bacterial virulence.
    Identify potential antibiotic targets and drug candidates.


